addons/account/tests/test_account_move_out_invoice.py

```diff
@@ -2472,14 +2474,18 @@ def test_out_invoice_change_period_accrual_1(self):
 
         accrual_lines = self.env['account.move'].browse(wizard_res['domain'][0][2]).line_ids.sorted('date')
         self.assertRecordValues(accrual_lines, [
-            {'amount_currency': 600.0,  'debit': 300.0, 'credit': 0.0,      'account_id': self.product_line_vals_1['account_id'],   'reconciled': False},
-            {'amount_currency': -600.0, 'debit': 0.0,   'credit': 300.0,    'account_id': wizard.revenue_accrual_account.id,        'reconciled': True},
-            {'amount_currency': 120.0,  'debit': 60.0,  'credit': 0.0,      'account_id': self.product_line_vals_2['account_id'],   'reconciled': False},
-            {'amount_currency': -120.0, 'debit': 0.0,   'credit': 60.0,     'account_id': wizard.revenue_accrual_account.id,        'reconciled': True},
+            {'amount_currency': 900.0,  'debit': 450.0, 'credit': 0.0,      'account_id': self.product_line_vals_1['account_id'],   'reconciled': False},
```

Comment:
weird. why does that change ?

Reply:
Now, I create all accrual lines and then, to avoid rounding issues, I sum all debit/credit to create the counterpart line.
It means I create now a move line of 90% in the first move instead of creating 2 lines 60% + 30%